### ISSUE TITLE
empty profile on julia-1.11

### ISSUE THREAD

**Issue by kalmarek** [timholy/ProfileView.jl#236]
running the README example I receive:

![Screenshot from 2024-10-25 16-45-16](https://github.com/user-attachments/assets/cf505c6e-02c0-4ccc-ac45-2f994794e1f5)

**Comment by fjebaker** [comment]
Same problem trying to run the example.
```
julia> versioninfo()
Julia Version 1.11.1
Commit 8f5b7ca12ad (2024-10-16 10:53 UTC)
Build Info:
  Official https://julialang.org/ release
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: 12 × 12th Gen Intel(R) Core(TM) i7-1255U
  WORD_SIZE: 64
  LLVM: libLLVM-16.0.6 (ORCJIT, alderlake)
Threads: 12 default, 0 interactive, 6 GC (on 12 virtual cores)
Environment:
  JULIA_EDITOR = code
  JULIA_NUM_THREADS = 
```

**Comment by fjebaker** [comment]
I see this is a known problem and that it has been addressed in https://github.com/JuliaLang/julia/issues/56327

**Comment by kalmarek** [comment]
This is an upstream issue, will be fixed in 1.11.2
